board/safety.h

```diff
@@ -49,6 +49,9 @@ int controls_allowed = 0;
 #include "safety/safety_gm.h"
 #include "safety/safety_ford.h"
 #include "safety/safety_cadillac.h"
+#ifdef PANDA
+#include "safety/safety_tesla.h"
```

Comment:
move this after https://github.com/commaai/panda/pull/132/files#diff-7b552a9c774a81e90fcf2183ff134492R47

Reply:
This is done and will be in my next commit.